### DISCUSSION TITLE
Coursera/Stanford Machine Learning Course using Candle

### DISCUSSION THREAD

**Discussion by vishpat** [huggingface/candle#1988] [Show and tell]
[Here](https://github.com/vishpat/candle-coursera-ml/tree/main) is the implementation of the ML algorithms (using Candle) taught in Andrew Ng's [Machine Learning Specialization](https://www.coursera.org/specializations/machine-learning-introduction) course on Coursera. Have created a [PR](https://github.com/huggingface/candle/pull/1989) to include this as an external resource.

**Comment by aminnasiri** [comment]
That is so great thank you
